parts/k8s/containeraddons/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -427,7 +426,7 @@ spec:
 # as the CNI plugins and network config on
 # each master and worker node in a Kubernetes cluster.
 kind: DaemonSet
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
I had to reassure myself that `apps/v1` went GA in Kubernetes 1.9, because AKS Engine still has to support 1.10 and later. :+1: